app/pages/home-logged-in.cjsx

```diff
@@ -60,6 +60,56 @@ FeaturedProjects = React.createClass
       <Link to="/projects" className="call-to-action standard-button x-large"><Translate content="home.featuredProjects.button" /></Link>
     </div>
 
+RecentProjects = React.createClass
+  displayName: 'RecentProjects'
+  
+  getInitialState: ->
+    projects: []
+    activityCounts: {}
+    title: ""
+    href: ''
+    action: ""
+  
+  componentWillMount: ->
+    @props.user
+      .get("project_preferences", page_size: 10, sort: '-updated_at')
```

Comment:
That would be useful here too: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/components/classifications-ribbon.cjsx#L100
rather than filtering that array later.